### ISSUE TITLE
`libc::abort` prevents `wasm` build

### ISSUE THREAD

**Issue by AlexArgoAi** [crossbeam-rs/crossbeam-channel#72]
Building for wasm currently fails because `libc::abort` is not available on this target.

```
$ rustup target add wasm32-unknown-unknown
$ cargo build --target wasm32-unknown-unknown
error[E0425]: cannot find function `abort` in module `libc`=======>    ] 17/18: crossbeam-channel                                                                                                                
   --> src/internal/channel.rs:268:28
    |
268 |             unsafe { libc::abort() }
    |                            ^^^^^ not found in `libc`
help: possible candidates are found in other modules, you can import them into scope
    |
3   | use std::intrinsics::abort;
    |
3   | use std::process::abort;
    |

error[E0425]: cannot find function `abort` in module `libc`
  --> src/internal/utils.rs:61:24
   |
61 |         unsafe { libc::abort() }
   |                        ^^^^^ not found in `libc`
help: possible candidates are found in other modules, you can import them into scope
   |
3  | use std::intrinsics::abort;
   |
3  | use std::process::abort;
   |

error: aborting due to 2 previous errors
```

We're pretty close to at least compiling! Do you think it would be possible to use an alternative to `libc::abort` in these cases?

**Comment by stjepang** [comment]
I've just replaced `libc::abort` with `std::process::abort` and released version `0.2.2`.

Can you try with the new version?

**Comment by AlexArgoAi** [comment]
It compiles! :tada: 
Real testing will be for another time.
Thanks a lot!
